apps/passport-client/components/shared/PCDCard.tsx

```diff
@@ -24,21 +24,40 @@ export function PCDCard({
   onClick?: () => void;
   hideRemoveButton?: boolean;
 }) {
+  const [state] = useContext(DispatchContext);
   const pcdPackage = usePackage(pcd);
   const displayOptions = useMemo(() => {
     if (pcdPackage?.getDisplayOptions) {
       return pcdPackage?.getDisplayOptions(pcd);
     }
   }, [pcd, pcdPackage]);
-  const header = isZuzaluIdentity
-    ? "VERIFIED ZUZALU PASSPORT"
-    : displayOptions?.header?.toUpperCase() ?? "PCD";
+
+  const visitorStatus = getVisitorStatus(state.self);
```

Comment:
all this visitor stuff only applies if `isZuzaluIdentity`, right? like, it shouldn't affect the ethereum PCD etc

Reply:
Yes, the fact that i had to made a change here is not ideal.